lua/entities/gmod_wire_addressbus.lua

```diff
@@ -59,7 +64,7 @@ function ENT:ReadCell(Address)
 			if self.Memory[i] then
 				if self.Memory[i].ReadCell then
 					self.DataBytes = self.DataBytes + 1
-					local val = self.Memory[i]:ReadCell(Address - self.MemStart[i])
+					local val = self.Memory[i]:ReadCell((Address+self.MemOffsets[i]) - self.MemStart[i])
```

Comment:
The parenthesis here and line 85 aren't needed

Reply:
I tend to be a bit overly paranoid about this kind of thing, but I've corrected it and added spaces